README.md

```diff
@@ -37,6 +37,29 @@ You should be able to provide your credentials when Matterfront starts up.
 
 Support for adding multiple teams through the UI is coming soon.
 
+## Electron and Chrome command-line args
+
+[Electron supports a subset of Chrome command-line switches](https://github.com/atom/electron/blob/master/docs/api/chrome-command-line-switches.md) when it starts up the main Chromium window. These command-line switches can be set via a config file in your home dir.
+
+Create a text file at `~/.matterfront/config.json`. (Where `~` is your home directory), and include a `chrome-args` key, like this:
+
+```json
+{
+  "chrome-args": {
+    "remote-debugging-port":"2929",
+    "no-proxy-server": ""
+  }
+}
+```
+
+Matterfront will read this file and apply each switch when it starts up the app. Note that config options that don't have a value can specify a value of `""`.
```

Comment:
What about setting a flags property on the chrome-args object, that takes
an array of flags?
On Dec 8, 2015 3:00 PM, "Chris Jaynes" <notifications@github.com> wrote:

> In README.md
> <https://github.com/lloeki/matterfront/pull/54#discussion_r47012751>:
>
> > +## Electron and Chrome command-line args
> > +
> > +[Electron supports a subset of Chrome command-line switches](https://github.com/atom/electron/blob/master/docs/api/chrome-command-line-switches.md) when it starts up the main Chromium window. These command-line switches can be set via a config file in your home dir.
> > +
> > +Create a text file at `~/.matterfront/config.json`. (Where `~` is your home directory), and include a `chrome-args` key, like this:
> > +
> > +```json
> > +{
> > +  "chrome-args": {
> > +    "remote-debugging-port":"2929",
> > +    "no-proxy-server": ""
> > +  }
> > +}
> > +```
> > +
> > +Matterfront will read this file and apply each switch when it starts up the app. Note that config options that don't have a value can specify a value of `""`.
>
> Sort of.
>
> I think true *does* work to set the no-proxy-server flag. I don't think
> false turns it back off. The reason I chose "" is because it matches how
> you'd pass the flags to the command line, where the docs say it looks like
> this:
>
> chrome --no-proxy-server
>
> Because the flags themselves are negatives, it's also a little awkward to
> imply that one might have to do:
>
> "no-proxy-server": false
>
> or
>
> "disable-http-cache": false
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lloeki/matterfront/pull/54/files#r47012751>.
>